molgenis/molgenis-emx2
# fix (dashboards): reorganised images in landing pages

What are the main changes you did:

The purpose of this PR is to reorganise all files stored in the `public` folder that are used in each of project specific Vue applications. This included the following changes.

1. Move images to `public/img` and everything else to `public/assets/` (if applicable)
2. Adjust the image paths in the vue applications
3. Very light typescript fixes where needed

The following apps were updated.

- [x] [cranio-provider](https://preview-emx2-pr-4374.dev.molgenis.org/DK1/cranio-provider/#/)
- [x] [cranio-public](https://preview-emx2-pr-4374.dev.molgenis.org/CranioPublic/cranio-public/#/)
- [x] [ern-genturis](https://preview-emx2-pr-4374.dev.molgenis.org/apps/ern-genturis/#/)
- [x] [ern-ithaca](https://preview-emx2-pr-4374.dev.molgenis.org/apps/ern-ithaca/#/)
- [x] [ern-reconnet](https://preview-emx2-pr-4374.dev.molgenis.org/apps/ern-reconnet/#/)
- [x] [ern-skin](https://preview-emx2-pr-4374.dev.molgenis.org/apps/ern-skin/#/)
- [x] [gportal](https://preview-emx2-pr-4374.dev.molgenis.org/apps/gportal/#/)
- [x] [imdhub-public](https://preview-emx2-pr-4374.dev.molgenis.org/apps/imdhub-public/#/)
- [x] [molgenis-viz](https://preview-emx2-pr-4374.dev.molgenis.org/data%20viz/molgenis-viz/#/docs/)

how to test:
- Navigate to the preview: https://preview-emx2-pr-4374.dev.molgenis.org
- View the following apps (listed above)
- Navigate the various pages to see if the images are displayed. This applies to header images, logos, other diagrams loaded into the text, etc.

> [!Warning]
> Some of these apps require specific schema configurations (e.g., linking multiple schemas together, adding reference databases, etc.). It isn't required for this PR as this mainly focuses on the images. As a result, you may see errors or warnings. These can be ignored. If anything is unclear, please leave a comment below.



todo:
